bindings/python/release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -ex
+
+# This script should be run on macOS to create the following distributions:
+# pymongocrypt-<version>.tar.gz
+# pymongocrypt-<version>-py2.py3-none-manylinux2010_x86_64.whl
+# pymongocrypt-<version>-py2.py3-none-macosx_10_9_x86_64.whl
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Build the source dist first
+rm -rf build
+python3.7 setup.py sdist
+
+# The libmongocrypt git revision release to embed in our wheels.
+REVISION=latest
+
+# Build the manylinux2010 wheel
+rm -rf ./libmongocrypt
+curl -O https://s3.amazonaws.com/mciuploads/libmongocrypt/rhel-62-64-bit/master/${REVISION}/libmongocrypt.tar.gz
+curl -O https://s3.amazonaws.com/mciuploads/libmongocrypt/rhel-62-64-bit/master/latest/libmongocrypt.tar.gz
+mkdir libmongocrypt
+tar xzf libmongocrypt.tar.gz -C ./libmongocrypt
+NOCRYPTO_SO=libmongocrypt/nocrypto/lib64/libmongocrypt.so
+chmod +x ${NOCRYPTO_SO}
+cp ${NOCRYPTO_SO} pymongocrypt/
+rm -rf ./libmongocrypt libmongocrypt.tar.gz
+
+docker run --rm -v `pwd`:/python quay.io/pypa/manylinux2010_x86_64 /python/build-manylinux-wheel.sh
```

Comment:
What advantage does building in manylinux2010 provide over just running python setup.py bdist_wheel? I assume it has something to do with auditwheel?

Reply:
One advantage is that it allows us to create the manylinux wheel on macOS. Another is that we don't need to install auditwheel. But most importantly, I believe it's required to use the container. See this quote from [PEP-571](https://www.python.org/dev/peps/pep-0571/#compilation-of-compliant-wheels) (emphasis mine):
> Like manylinux1, the auditwheel tool adds manylinux2010 platform tags to linux wheels built by pip wheel or bdist_wheel **in a manylinux2010 Docker container**.

Reply:
Cool. Thanks for clarifying.